app/app/urls.py

```diff
@@ -61,6 +62,13 @@
 urlpatterns = [
     path('jsi18n/', JavaScriptCatalog.as_view(), name='javascript-catalog'),
 
+    # inbox
+    path('api/v0.1/notifications/', inbox.views.notifications, name='notifications'),
```

Comment:
We should either migrate these to DRF or move them out of the API namespacing.

If you opt to move them, please follow the `kudos` and `avatars`: `urls.py` as a guide and `include` it here namespaced as `inbox` (or rename the django module to `notifications`)